microcosm_daemon/health_reporter.py

```diff
@@ -0,0 +1,57 @@
+import os
+from logging import getLogger
+
+from microcosm.api import defaults, typed
+
+from microcosm_daemon.error_policy import ExitError
+
+
+try:
+    import requests
+except ImportError:
+    requests = None  # type:ignore
+
+
+logger = getLogger("daemon.health_reporter")
+
+
+class HealthReporter:
+    def __init__(self, graph):
+        self.healthcheck_server_host = graph.config.health_reporter.healthcheck_server_host
+        self.healthcheck_server_port = graph.config.health_reporter.healthcheck_server_port
+
+    def __call__(self, health, prev_health, errors):
+        self.heartbeat()
```

Comment:
Note that `heartbeat` is not taking in `health` as argument.
The reason is that, as things currently stand in microcosm-pubsub, `health` will be non-0 if any errors happened during processing of a pubsub message. In most cases however, that doesn't mean the daemon itself is unhealthy however – and figuring out the cases in which it does mean it's unhealthy is tricky.
For that reason, for now we're going with a "dumb" heartbeat.